docs/baremetal_example.md

```diff
@@ -0,0 +1,5 @@
+## Bare metal example
+
+To start using the bare metal profile on Mbed OS 5, please see the blinky bare metal example:
```

Comment:
How about changing this sentence to - "To learn about using the bare metal profile on Mbed OS 5,... ". 

Everything else looks great.